ui/tsconfig.json

```diff
@@ -20,7 +20,7 @@
     "module": "es2020",
     "noImplicitOverride": true,
     "lib": [
-      "es2018",
+      "es2022",
```

Comment:
lib es version is bumped to es2022 to use `.flat()`.

Reply:
generally target es version is lower than lib es version to compatibility.
but this project target es is higher than lib es version. ??
anyway, it is not change of target es version, so I think no effect browser compatibility.